### PR TITLE
🌐 Add ability for amp-story-auto-ads CTA strings to be localized

### PR DESCRIPTION
**Pull request by newmuis**
Dependent on #21239.

Note that this PR does not address the "Ad" attribution label; that will need to come in a followup PR (I missed it when looking for strings that need translation 😦)

### CODE REVIEW

File: extensions/amp-story-auto-ads/0.1/_locales/en.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline extensions/amp-story-auto-ads/0.1/_locales/en.js]: nit: 2019

**Comment by calebcordry** [comment]
cc/ @jasti

File: extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js [hunk 2]
```diff
@@ -215,6 +221,17 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
      * @private {?../../amp-story/0.1/amp-story-store-service.AmpStoryStoreService|?../../amp-story/1.0/amp-story-store-service.AmpStoryStoreService}
      */
     this.storeService_ = null;
+
+    /** @private @const {!LocalizationService} */
+    this.localizationService_ = new LocalizationService(this.win);
```
Comment (gmajoulet) [inline extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js]: Should we have a service getter (eg: `Services.localizationServiceFor(this.win)`) that could be shared by multiple extensions instead?

Comment (newmuis) [reply extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js]: We can; let me send in a followup refactoring of localization more generally.  I'll file an issue, as there are a number of things that are not ideal.

**Pull request merged**